src/translations/locales/en.json

```diff
@@ -56,6 +56,11 @@
   "alarmtemplates_new.characters": "characters",
   "alarmtemplates_new.create_template": "Create template",
   "alarmtemplates_new.sms_max_char_warning": "SMS messages have a limit of 160 characters after substituting the parameter tags",
+  "authorization.redirected_based_onrole": "You do not have the rights to acces this data under the selected organisation. \nYou will be redirected",
+  "authorization.requires_role": "Requires role: ",
+  "authorization.requires_roles": "Requires one of the following roles: ",
+  "authorization.organisation_not_allowed_current_page": "! Organisation not authorized to visit current page !",
+  "authorization.no_roles_message": "Dear user, \nYou seem not to be in any organisations that can acces the management pages. \nTherefore you are redirected to the mainpage",
```

Comment:
Acces -> access. Mainpage -> main page

Reply:
Also we are very inconsistent with s/z usage, like authorization, organisation...

Reply:
access changed.
Inconsistencies in organisation authorization -> can I fix this in a new pr?
This is already a problem. Not introduced in this pr